[IMP] Administration: Update outdated info & screenshots (#7820)
odoo/documentation: content/administration/maintain/on_premise.rst

* Do you have a valid Enterprise subscription?

>   * Check if your subscription details have the tag "In Progress" on

Reviewer: ```suggestion
  * Check if your subscription details have the tag :guilabel:`In Progress` on
```
We use the [guilabel markup](https://www.odoo.com/documentation/16.0/contributing/documentation/rst_cheat_sheet.html#gui-element) when we identify a specific UI element.

    In this case, you need to change the UUID on your test databases to solve this
    issue. You will find more information about this in :ref:`this section <duplicate_premise>`.

    For your information, we identify each database with a UUID. Therefore, each
    database should have a distinct UUID to ensure that registration and invoicing
    proceed effortlessly for you and us.

* Check your network and firewall settings

>   * The Update notification must be able to reach Odoo’s subscription validation servers. In other

Reviewer: ```suggestion
  * The Update notification must be able to reach Odoo's subscription validation servers. In other
```

    words, make sure that the Odoo server can open outgoing connections towards:

    * services.odoo.com on port 443 (or 80)
    * services.openerp.com on port 443 (or 80) for older deployments
  * Once you activate your database, you must keep these ports open, as the Update notification
    runs once a week.

Error message due to too many users
-----------------------------------

If you have more users in your local database than provisioned in your Odoo Enterprise subscription,
you may encounter this message:

.. image:: on_premise/add_more_users.png
   :align: center
   :alt: This database will expire in X days, you have more users than your subscription allows

When the message appears you have 30 days before the expiration. The countdown is updated every day.

Solutions
~~~~~~~~~

- **Add more users** to your subscription: follow the link and validate the upsell quotation and pay
  for the extra users.

> - **Deactivate users** as explained in this `documentation <documentation.html#deactivating-users>`_

Reviewer 2: This link is outdated

> Database expired error message

Reviewer 2: The force ping section link above refers to an empty section

------------------------------

> If your database reaches its expiration date before you renew your subscription,

Reviewer: ```suggestion
If your database expires before you renew your subscription,
```
Suggestion to simplify (the less wordy, the better)

you will encounter this message:

.. image:: on_premise/database_expired.png
   :align: center
   :alt: This database has expired.

This **blocking** message appears after a non-blocking message that lasts 30 days. If

> you fail to take action before the end of the countdown, the database will expire.

Reviewer: ```suggestion
you fail to act before the end of the countdown, the database will expire.
```

Solutions
~~~~~~~~~

> * Renew your subscription: Follow the link and renew your subscription - note that if you wish to
>   pay by Wire Transfer, your subscription will effectively be renewed only when the payment arrives,
>   which can take a few days. Credit card payments are processed immediately.

Reviewer: ```suggestion
* Renew your subscription: follow the link and renew your subscription. If you
  pay by wire transfer, your subscription will be renewed when the payment arrives,
  which can take a few days. Credit card payments are processed immediately.
```

* Contact our `Support <https://www.odoo.com/help>`__

> .. _force_ping:

Reviewer 2: Section is empty

====================

You can duplicate your database by accessing the database manager on your

> server (<odoo-server>/web/database/manager). On this page, you can easily

Reviewer: ```suggestion
server (<odoo-server>/web/database/manager). On this page, you can
```
We stick to facts, adding "easily" isn't necessary :) (I know it wasn't you ;))

duplicate your database (among other things).

.. image:: on_premise/db_manager.gif
   :align: center

> When you duplicate a local database, it is **strongly** advised to change the duplicated
> database’s UUID (Unniversally Unique Identifier), since this UUID is how your database identifies
> itself with our servers. Having two databases with the same UUID could result in invoicing problems
> or registration problems down the line.

Reviewer: ```suggestion
```
I don't think it's necessary to say that since Odoo automatically changes the UUID of duplicated database since Odoo 9

.. note::
   From July 2016 onward, Odoo 9  and up now automatically changes the UUID of a duplicated database;

>    a manual operation is no longer required.

Reviewer: Not necessary either

> The database UUID is currently accessible from the menu :menuselection:`Settings --> Technical -->
> System Parameters`, we advise you to use a `UUID generator <https://www.uuidtools.com>`_ or to
> use the Unix command ``uuidgen`` to generate a new UUID. You can then simply replace it like any
> other record by clicking on it and using the edit button.
> 

Reviewer: ```suggestion
```
I don't think it's necessary either?

> .. image:: on_premise/db_uuid.png

Reviewer: I'm not sure this section is worth keeping, but FYI, the screenshot is way too wide, making it difficult to view on most screens. A crop like this would be sufficient:
<img width="537" alt="firefox_ziRWTLTZbO" src="https://github.com/odoo/documentation/assets/70884563/725839d4-b669-4a06-8e96-932149ac9673">

> ====================

Reviewer 2: This part explains how to duplicate a database for production purpose but most of the time customers will want to duplicate a database to do some testing like on saas or sh. To do so they can check the checkbox neutralize when making a duplicate via <odoo-server>/web/database/manager
This will execute all the neutralize.sql scripts for every installed module. Here we should link to the documentation page about neutralize which I think do not exist (feature introduced in 16: https://github.com/odoo/odoo/pull/102792)

You can duplicate your database by accessing the database manager on your
server (<odoo-server>/web/database/manager). On this page, you can
duplicate your database (among other things).

> .. image:: on_premise/db_manager.gif

Author: I remade the GIF but it is 800KB versus the 88 of the original. I cannot use pngquant because this is not a PNG. What do you recommend?

> .. image:: on_premise/db_manager.gif

Reviewer 2: Hello I usually use vokoscreen to record a video (should be installed on your linux computer)
Then using ffmpeg I convert it to gif using this command: 
`ffmpeg -ss 0 -t 5 -i input.mp4 -vf "fps=10,scale=320:-1:flags=lanczos,split[s0][s1];[s0]palettegen[p];[s1][p]paletteuse" output.gif`

-ss 0 : change 0 by the number of seconds you want to skip in the video
-t 5 change 5 by the time the gif should last in seconds
fps=10 you can increase/decrease the frame per second in the gif (more frames more weight)
scale=320 you can increase/decrease the resolution of the gif (higher resolution more weight)

That being said is it really necessary to have a gif there ?
Just 2 screenshot one of the database list and one of the popup should be enough.
